src/style.css

```
@@ -0,0 +1,55 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  width: 0;
+  height: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.container {
+  position: absolute;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+}
+
+.stars > :not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--5 .stars__star {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--0 .stars__star {
```

combine selectors with coma

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url("images/star-active.svg");
}
```
and so on

---

thank you for your comments